tests/TaskLineRenderer.test.ts

```diff
@@ -31,13 +32,21 @@ const fieldRenderer = new TaskFieldRenderer();
  * @param layoutOptions for the task rendering. Skip for default options. See {@link LayoutOptions}.
  *
  * @param testRenderer imitates Obsidian rendering. Skip for the default {@link mockTextRenderer}.
+ *
+ * @param queryLayoutOptions for the task rendering. Skip for default options. See {@link QueryLayoutOptions}.
  */
-async function renderListItem(task: Task, layoutOptions?: LayoutOptions, testRenderer?: TextRenderer) {
+async function renderListItem(
+    task: Task,
+    layoutOptions?: LayoutOptions,
+    testRenderer?: TextRenderer,
+    queryLayoutOptions?: QueryLayoutOptions,
```

Comment:
A small thing, butt it would be a bit tidier if the testRenderer was the last argument

Reply:
Good point =)